### PR TITLE
Current Terminal size added in About popup

### LINKED ISSUE #1536: Add current terminal window size to About popup
### Description of suggested feature

It would be useful during reporting issues to know the size of the current terminal.

This is already available at runtime via urwid, since the popup code relies on it to determine which width (and maximum width) to use.

This entry should appear at the bottom of the 'Detected Environment' section of the 'About' popup.

Since the terminal size can often be changed while a popup is open, we will likely want to call this 'Current terminal size', though this can be adjusted later during a PR or after merging.

### PR DESCRIPTION
**Pull request by Swarnim114**
closes #1536 

### What does this PR do, and why?
Adds a Terminal size label under the "Detected Environment " in About popup

image

### External discussion & connections

- [ ] Discussed in **#zulip-terminal** in `topic`
- [x] Fully fixes #1536 
- [ ] Partially fixes issue #
- [ ] Builds upon previous unmerged work in PR #
- [ ] Is a follow-up to work in PR #
- [ ] Requires merge of PR #
- [ ] Merge will enable work on #

### How did you test this?

- [x] Manually - Behavioral changes
- [x] Manually - Visual changes
- [x] Adapting existing automated tests
- [ ] Adding automated tests for new behavior (or missing tests)
- [ ] Existing automated tests should already cover this (*only a refactor of tested code*)

### Self-review checklist for each commit
- [ ] It is a [minimal coherent idea](https://github.com/zulip/zulip-terminal#structuring-commits---speeding-up-reviews-merging--development)
- [ ] It has a commit summary following the [documented style](https://github.com/zulip/zulip-terminal#structuring-commits---speeding-up-reviews-merging--development) (title & body)
- [ ] It has a commit summary describing the  motivation and reasoning for the change
- [ ] It individually passes linting and tests
- [ ] It contains test additions for any new behavior
- [ ] It flows clearly from a previous branch commit, and/or prepares for the next commit

### Visual changes    
image

### CODE REVIEW

**Comment by Swarnim114** [comment]
@zulipbot  label "PR needs review"

**Comment by Swarnim114** [comment]
Hi @neiljp,

Sorry for the mess-up in that PR earlier. I'm new to open source development. Thanks for your understanding, and please continue guiding me.

File: zulipterminal/ui_tools/views.py [hunk 1]
```diff
@@ -1076,6 +1076,9 @@ def __init__(
         super().__init__(controller, widgets, "EXIT_POPUP", width, title)
 
 
+# views.py
+
+
```
Comment (rsashank) [inline zulipterminal/ui_tools/views.py:1081]: Is there a specific reason for these additions?

Other than that, LGTM

Comment (Swarnim114) [reply zulipterminal/ui_tools/views.py:1081]: ahh no. sorry !
they are there bye mistake

Comment (Swarnim114) [reply zulipterminal/ui_tools/views.py:1081]: @rsashank  can u please guide me on how to remove them and still keep the commits to only one.

**Comment by Swarnim114** [comment]
@zulipbot   label "PR ready to be merged"

File: zulipterminal/ui_tools/views.py (hunk 1 above)
Comment (rsashank) [reply zulipterminal/ui_tools/views.py]: You can make the change and squash it into your first commit. Alternatively, you could rebase and edit the commit.

Look into [interactive rebase](https://docs.github.com/en/get-started/using-git/using-git-rebase-on-the-command-line).

**Pull request closed without merging**